pdp/config.env

```diff
@@ -0,0 +1,17 @@
+APP_ROOT=http://tools.pacificclimate.org/dataportal
+DATA_ROOT=http://tools.pacificclimate.org/dataportal/data
+TITLE=
+ENSEMBLE_NAME=
+# add password for production
+DSN=postgresql://user:pass@host/database
+# add password for production
+PCDS_DSN=postgresql://user:pass@host/database
+JS_MIN=false
+# change for development
+GEOSERVER=URLhttp://tools.pacificclimate.org/geoserver/
```

Comment:
Typo?

Reply:
I'm going to blame emacs... must have hit some "swap words" macro accidentally.